content/_index.en.md

```diff
@@ -0,0 +1,106 @@
+# Sunny tank? Super simple.
+
+evcc optimises sustainability when charging electric vehicles.
+It enables charging using as much self-generated power as possible.
+And best of all, it can do so in most cases without any further modification to your house's existing electrical installation.
+
+---
+
+{{< live-telemetry >}}
+
+---
+
+## What makes evcc special?
+
+- Charge your EV with your excess solar power
+- Use your existing systems (solar and battery inverters, wallboxes, energy meters, etc.)
+- Intuitive, clean, and responsive user interface
+- 100 % Open Source
+
+---
+
+## Responsive Interface
+
+{{< screenshots class="content-gallery" >}}
+{{< screenshot-entry src="1_evcc_mobile_standard" width="508" >}}
+{{< screenshot-entry src="2_evcc_mobile_details" width="508" >}}
+{{< screenshot-entry src="3_evcc_mobile_savings" width="508" >}}
+{{< screenshot-entry src="4_evcc_tablet_standard" width="1387" >}}
+{{< screenshot-entry src="5_evcc_tablet_details" width="1387" >}}
+{{< /screenshots >}}
+
+---
+
+## How EVCC works
+
+{{< full_width_image src="img/evcc-schema.svg" alt="How it works" width="200" height="100" >}}
```

Comment:
Making a translated version of this SVG would be really easy to implement, but the one shipped as standard has the text as shapes. Is there an original we can use?

Reply:
I've replaced the file with a version that uses text natively. Since I think the Germans will be able to understand the English labels as well I've decided not to localize/double the file.